### ISSUE TITLE
Control being interrupted when play another url while the control loading another audio

### ISSUE THREAD

**Issue by phanbaohuy96** [muslimtv/flutter_playout#6]
The issues happened when I fast-playing a few audios. Because I want to load the audio first, then I play another audio that issue has happened. I think we should have a function to load the audio instead of using the 'play' function.

**Comment by KhuramKhalid** [comment]
Yes this is currently a limitation in the player implementation. Multiple players cannot be created at the same time and so loading URL on `play` is a way to enforce that. I will look into finding a better implementation where we can support multiple players.

**Comment by phanbaohuy96** [comment]
In my case, i dont need multiple players, that is happens when a play a audio when the player loading another one. I think i should clear that one and load new audio. I hope this case will be resolved and hope you find the best way to use multiple players.
